src/scripts/utils.js

```diff
@@ -59,3 +71,21 @@ export const toWholeCoins = satoshis => {
 export const getCurrencyName = symbol => {
   return symbol === 'BTC' ? 'Bitcoin' : 'Litecoin';
 };
+
+// TODO: refactor how we copy
```

Comment:
You can remove this TODO, its irrelevant.

Reply:
Sure? Why did you put it there in the first place?

Reply:
The implementation was off the top of my head at first so I wasn't sure if it would cover all use cases, but I don't think we need anything fancier than that.